docs/COMMANDS.md

```diff
@@ -29,7 +29,19 @@ This will run a server that's accessible in the entire local network and shows t
 ## Unit testing
 
 ```Shell
+$ npm run lint
+```
+
+Checks .js files for any style errors.
+
+```Shell
 $ npm run test
 ```
 
-Tests your application with the unit tests specified in the `test` folder.
+Lints .js file and tests your application with the unit tests specified in the `test` folder.
+
+```Shell
+$ npm run test:watch
+```
+
+Watches and run tests on .js file changes. To be run on a separate terminal.
```

Comment:
Maybe change it to something to the effect of `Runs unit tests everytime you change something in your JavaScript code`?